.local-scripts/gather-dpkg.sh

```diff
@@ -29,6 +29,25 @@ if [ "${#packages[@]}" -eq 0 ]; then
 	exit 1
 fi
 
+if [ -e /etc/apt/sources.list ]; then
```

Comment:
At some point this is going to need to also check for `/etc/apt/sources.list.d` (https://github.com/debuerreotype/debuerreotype/pull/128), so we might as well add that now:

```suggestion
if [ -e /etc/apt/sources.list ] || [ -d /etc/apt/sources.list.d ]; then
```